docs/CONTRIBUTING_DEVELOPMENT.md

```diff
@@ -0,0 +1,262 @@
+# Contribution process for developers
+
+If you plan on contributing code to Clarity, please follow this step-by-step
+process to reduce the risk of significant changes being requested when you
+submit your pull request. We'll work with you on every step and try to be as
+responsive as possible.
+
+## I. Proposal template
+
+Before you start coding anything, please fill out the following proposal
+template in as much detail as you can. The more complete your details, the
+better, but not all questions will apply for every change. As you fill this out,
+please make sure to follow our coding guidelines for components.
+
+* [Angular Guidelines](/docs/CODING_GUIDELINES_ANGULAR.md)
+* [Core Web Component Guidelines](/docs/CODING_GUIDELINES_CORE.md)
+
+```markdown
```

Comment:
Do we want ` ```markdown ` here?

Reply:
Yes, it renders the contribution content that the user should copy into a new issue